src/status_im/ui/screens/accounts/recover/events.cljs

```diff
@@ -27,14 +28,16 @@
     (let [data (json->clj result)
           public-key (:pubkey data)
           address (:address data)
+          phrase     (signing-phrase/generate)
           {:keys [public private]} keypair
           account {:public-key          public-key
                    :address             address
                    :name                (generate-gfy public-key)
                    :photo-path          (identicon public-key)
                    :updates-public-key  public
                    :updates-private-key private
-                   :signed-up?          true}]
+                   :signed-up?          true
+                   :signing-phrase      phrase}]
```

Comment:
spec?

Reply:
That I completely missed. Thank you!

Reply:
that's strange you should see red screen with error in dev mode, hm

Reply:
Well, that's a cool reminder, but I didn't. It just worked.